mysite/polls/models.py

```diff
@@ -1,11 +1,12 @@
 import datetime
 from django.db import models
 from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
 
 
 class Question(models.Model):
     question_text = models.CharField(max_length=200)
```

Comment:
Why this is not translated ?

Reply:
Are you talking about the created questions?

Reply:
question_text field is not translated